cmd/influx/pkg.go

```diff
@@ -816,11 +817,12 @@ func (b *cmdPkgBuilder) registerPkgPrintOpts(cmd *cobra.Command) {
 }
 
 func (b *cmdPkgBuilder) registerPkgFileFlags(cmd *cobra.Command) {
-	cmd.Flags().StringSliceVarP(&b.files, "file", "f", nil, "Path to template file")
+	cmd.Flags().StringSliceVarP(&b.files, "file", "f", nil, "Path to template file; A https scheme will provide a remote file and a file scheme or missing scheme will be interpreted as a file")
```

Comment:
~A https scheme will provide a remote file and a file scheme or missing scheme will be interpreted as a file~
Supports HTTP(S) URLs or file paths.